saleor/static/dashboard-next/orders/components/OrderList/OrderList.tsx

```diff
@@ -56,6 +56,9 @@ const styles = (theme: Theme) =>
     },
     link: {
       cursor: "pointer"
+    },
+    relative: {
```

Comment:
I'm afraid that `relative` class name can do the `position: absolute` thing, this is not the UI library and we should use more descriptive class names, like `table` I'd use here